Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rails:4.2.6
+
+RUN echo "deb http://us.archive.ubuntu.com/ubuntu precise main universe" | tee -a /etc/apt/sources.list
```

Comment:
What are you adding this source for? I don't see any `apt-get` commands being run. Also, it seems weird to add an Ubuntu apt source onto a Debian-based image.